stac/src/main/scala/StacLink.scala

```diff
@@ -8,24 +8,27 @@ case class StacLink(
     href: String,
     rel: StacLinkType,
     _type: Option[StacMediaType],
-    title: Option[String]
+    title: Option[String],
+    labelExtAssets: List[String]
```

Comment:
Do you think we probably should make this field `labelExtAssets: Option[List[String]]`?

Reply:
Update: based on an off-line discussion, `labelExtAssets: List[String]` should suffice at this stage.